src/main/java/domain/RankRecord.java

```diff
@@ -10,7 +10,16 @@
 
     public RankRecord() {
         recordPool = new HashMap<>();
+        initRecordPool();
+    }
+
+
+    public RankRecord(Map<Rank, Integer> pool) {
+        recordPool = pool;
+        initRecordPool();
+    }
```

Comment:
중복되는 생성자 코드는 아래처럼 정리할 수 있습니다 :)
```suggestion
    public RankRecord() {
        this(new HashMap<>());
    }


    public RankRecord(Map<Rank, Integer> pool) {
        recordPool = pool;
        initRecordPool();
    }
```

Reply:
네 더 낫네요! 😊